src/triggers/Prevent_Delete.trigger

```diff
@@ -0,0 +1,7 @@
+trigger Prevent_Delete on Timesheet__c (before delete) {
+    for(Timesheet__c t : Trigger.old){
+        if(t.Submitted__c == true){
+            t.addError('Cannot delete a submitted timesheet'); 
+        }
+    }
```

Comment:
You shoud have only one trigger per object, which then holds all the required  events.